WinUIGallery/ControlPages/ColorPaletteResourcesPage.xaml

```diff
@@ -1,12 +1,16 @@
 <Page
     x:Class="WinUIGallery.ControlPages.ColorPaletteResourcesPage"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:local="using:WinUIGallery"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:WinUIGallery"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:localControls="using:WinUIGallery.DesktopWap.Controls"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" mc:Ignorable="d">
 
     <StackPanel>
-        <Image Source="https://github.com/Microsoft/fluent-xaml-theme-editor/raw/c27345fe37debcdfcbdd578ecb01e0715dd36ccd/XamlThemeEditor_screenshot.png" />
+        <localControls:SampleThemeListener>
+            <Image Source="https://github.com/Microsoft/fluent-xaml-theme-editor/raw/c27345fe37debcdfcbdd578ecb01e0715dd36ccd/XamlThemeEditor_screenshot.png" />
```

Comment:
@marcelwgn 
Wouldn't it be better to include this image in the assets rather than using an online link, ensuring the user can view it even when offline?

Reply:
You are completely correct, however that image is already out of date so I would hold of on that until there is an more up to date image for this

Reply:
Agreed